external/llvm-project/mlir/lib/Conversion/GPUToROCDL/LowerGpuOpsToROCDLOps.cpp

```diff
@@ -68,10 +68,10 @@ static bool canBeCalledWithBarePointers(gpu::GPUFuncOp func) {
 
 static constexpr StringLiteral amdgcnDataLayout =
     "e-p:64:64-p1:64:64-p2:32:32-p3:32:32-p4:64:64-p5:32:32-p6:32:32"
-    "-p7:160:256:256:32-p8:128:128-i64:64-v16:16-v24:32-v32:32-v48:64-v96:"
-    "128-v192:256-v256:256-v512:512-v1024:1024-v2048:2048-n32:64-S32-A5-"
-    "G1-ni:7:8";
-    
+    "-p7:160:256:256:32-p8:128:128-p9:192:256:256:32-i64:64-v16:16-v24:32-v32:"
+    "32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024-v2048:2048-n32:"
+    "64-S32-A5-G1-ni:7:8:9";
+
```

Comment:
Why this change here?

Reply:
I did not change this file initially.  I had a git-clang failure because of a whitespace change.  When I synced that, I had a test failure because this layout variable changed on the develop branch while I still had the old version.

Reply:
Yes I think if you merge the develop into your branch -- even when your branch dont have conflicts with develop -- this wont be needed because clang-format checks against origin/develop and if your branch is upto date with origin/develop clang-format wont even touch this file.